pages/_en-US/other/godmode9i.md

```diff
@@ -6,20 +6,20 @@ title: GodMode9i
 description: Information about GodMode9i
 ---
 
-[GodMode9i](https://github.com/DS-Homebrew/GodMode9i) is a file browser for the Nintendo DS ecosystem, taking inspiration from GodMode9 for the Nintendo 3DS. Dump cartridges, manage SD card files, browse NitroFS and more.
+[GodMode9i](https://github.com/DS-Homebrew/GodMode9i/) is an open source file browser for the Nintendo DS ecosystem, taking inspiration from GodMode9 for the Nintendo 3DS.
+GodMode9i has the ability to dump game cartridges, manage SD card files, browse NitroFS, calculate the SHA1 hash of files, edit binary files with a hex editor, and more.
```

Comment:
Do a double newline when you are adding an extra line. Otherwise it doesn't appear as two distinct lines and gets rendered as a normal space. ~~just markdown things~~

Reply:
This was on purpose, since the original text had it as one line (I just split it up since the markdown looked cluttered).

Reply:
Personally, even if it looks cluttered, I'd keep it on one line. I get what you're trying to say, though.

Reply:
Alright, I'll keep it on one line then :p